lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -2703,6 +2703,7 @@ llvm::DIType *CGDebugInfo::CreateTypeNode(QualType Ty, llvm::DIFile *Unit) {
   case Type::Decltype:
   case Type::UnaryTransform:
   case Type::PackExpansion:
+  case Type::TypeVariable:
```

Comment:
Should it be mapped the same way as "void" or "void *"? I think it should be mapped same as "void." I'll look more into it and will definitely have to fix it, because otherwise I cannot use clang-cl to build benchmarks. I am currently getting an error building zlib when I use type variables in the code, and this seems to be the cause of the problem.